src/lib.rs

```diff
@@ -16,7 +16,6 @@
 //! For `#[nserde(..)]` supported attributes for each format check [Features support matrix](https://github.com/not-fl3/nanoserde#features-support-matrix)
 
 #![cfg_attr(not(feature = "std"), no_std)]
-#![cfg_attr(not(feature = "std"), feature(error_in_core))]
```

Comment:
This will set our msrv for `no_std` to 1.81, but I think it's worth it so that nightly isn't a concern any more

Reply:
> This will set our msrv for `no_std` to 1.81, but I think it's worth it so that nightly isn't a concern any more

I agree, 0.2 will be a good moment to push msrv!